msobooking.yaml

```diff
@@ -0,0 +1,436 @@
+      person.ageatevent:
+        type: "person.ageatevent"
+        column: "AGE"
+    name: "persondetails"
+
+  persondetailscriminaljustice:
+    fqn: "ol.persondetailscriminaljustice"
+    entitySetName: "MSOPersondetailscriminaljustice"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "SENTENCED_AS_ADULT"
+      criminaljustice.treatasadult:
+        type: "criminaljustice.treatasadult"
+        column: "SENTENCED_AS_ADULT"
+    name: "persondetailscriminaljustice"
+
+associationDefinitions:
+  has:
+    fqn: "ol.has"
+    entitySetName: "MSOHas"
+    src: "person"
+    dst: "persondetails"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "assn_has"
+    name: "has"
+
+  has2:
+    fqn: "ol.has"
+    entitySetName: "MSOHas"
+    src: "person"
+    dst: "persondetailscriminaljustice"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "assn_has2"
+    name: "has2"
+
+  sentencedwith:
+    fqn: "ol.sentencedwith"
+    entitySetName: "MSOSentencedWith"
+    src: "person"
+    dst: "sentence"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "assn_sentencedwith"
+    name: "sentencedwith"
+
+  resultsin:
+    fqn: "ol.resultsin"
+    entitySetName: "MSOResultsIn"
+    src: "sentence"
+    dst: "jailstay"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "assn_resultsin"
+    name: "resultsin"
+
+  subjectof:
+    fqn: "ol.subjectof"
+    entitySetName: "MSOSubjectof"
+    src: "person"
+    dst: "jailstay"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "assn_subjectof"
+    name: "subjectof"
+
+  subjectof2:
+    fqn: "ol.subjectof"
+    entitySetName: "MSOSubjectof"
+    src: "person"
+    dst: "dnatest"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "assn_subjectof2"
+    name: "subjectof2"
+
+  reported:
+    fqn: "ol.reported"
+    entitySetName: "MSOReported"
+    src: "person"
+    dst: "immigrationstatus"
+    propertyDefinitions:
+      general.id:
+        type: "general.id"
+        column: "assn_reported"
+    name: "reported"
+
+  involvedin:
+    fqn: "ol.involvedin"
+    entitySetName: "MSOInvolvedin"
+    src: "officer"
+    dst: "jailstay"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "assn_involvedin"
+    name: "involvedin"
+
+  involvedin2:
+    fqn: "ol.involvedin"
+    entitySetName: "MSOInvolvedin"
+    src: "releaseofficer"
+    dst: "jailstay"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "assn_involvedin2"
+    name: "involvedin2"
+
+  involvedin3:
+    fqn: "ol.involvedin"
+    entitySetName: "MSOInvolvedin"
+    src: "committingofficer"
+    dst: "jailstay"
+    propertyDefinitions:
+      ol.id:
+        type: "ol.id"
+        column: "assn_involvedin3"
+    name: "involvedin3"
+
+  locatedat:
+    fqn: "ol.locatedat"
```

Comment:
I don't think this is strictly needed

Reply:
This is also a not very useful column of data - maybe we should talk to Sonya about the possibility of removing it, however it does have some information which seems unneccesary to disclude - the second top value after MDX is ESX - 142 (1/1000 rows)

Reply:
We should definitely include committing county, just not sure we should tie the committing officer to it. S/he is already tied to the jailstay.